core/src/main/java/org/apache/brooklyn/core/mgmt/ha/OsgiArchiveInstaller.java

```diff
@@ -336,14 +358,14 @@ private synchronized void close() {
             if (!updating) { 
                 osgiManager.managedBundlesRecord.addManagedBundle(result);
                 result.code = OsgiBundleInstallationResult.ResultCode.INSTALLED_NEW_BUNDLE;
-                result.message = "Installed "+result.getMetadata().getVersionedName()+" with ID "+result.getMetadata().getId();
+                result.message = "Installed Brooklyn catalog bundle "+result.getMetadata().getVersionedName()+" with ID "+result.getMetadata().getId()+" ["+result.bundle.getBundleId()+"]";
```

Comment:
I'm getting this line being hit every time I stop and restart Karaf; the effect is that the default catalog bom gets added as a new bundle every time,

    karaf@brooklyn()> list -s | grep brooklyn-catalog-bom
    284 | Active   |  80 | 0.12.0.SNAPSHOT          | brooklyn-catalog-bom-kmJluM62                                                                                                                                                                                                                                                                   | brooklyn-catalog-bom-kmJluM62
    286 | Active   |  80 | 0.12.0.SNAPSHOT          | brooklyn-catalog-bom-Rx1SYA2u                                                                                                                                                                                                                                                                   | brooklyn-catalog-bom-Rx1SYA2u
    287 | Active   |  80 | 0.12.0.SNAPSHOT          | brooklyn-catalog-bom-WntzbKnZ                                                                                                                                                                                                                                                                   | brooklyn-catalog-bom-WntzbKnZ

 and you get another copy of its jar in the persisted `bundles` directory.